Normalize.lua

```diff
@@ -143,3 +143,10 @@ function Normalize:type(type, tensorCache)
   end
   return self
 end
+
+function Normalize:clearState()
+  self._output = nil
```

Comment:
there are a few more buffers that need to be cleaned
* [`buffer`](https://github.com/szagoruyko/nn/blob/clearState/Normalize.lua#L20)
* [`norm`](https://github.com/szagoruyko/nn/blob/clearState/Normalize.lua#L19)
* [`normp`](https://github.com/szagoruyko/nn/blob/clearState/Normalize.lua#L34)
* [`cross`](https://github.com/szagoruyko/nn/blob/clearState/Normalize.lua#L62)